src/sdp.c

```diff
@@ -111,27 +114,32 @@ int sdp_decode_multipart(const struct pl *ctype_prm, struct mbuf *mb)
 	if (err)
 		return err;
 
-	if (re_snprintf(expr, sizeof(expr), "--%r[^]+", &bnd) < 0)
-		return ENOMEM;
-
-	/* find 1st boundary */
-	err = re_regex((char *)mbuf_buf(mb), mbuf_get_left(mb), expr, &s);
+	err = pl_strcpy(&bnd, bnd_str, sizeof(bnd_str));
 	if (err)
 		return err;
 
+	err = mbuf_strdup(mb, &buf, mbuf_get_left(mb));
+	if (err || !buf)
```

Comment:
both checks? I guess that `if (err)` is enough

Reply:
True, removed.